tutorials/high_performance_networking/README.md

```diff
@@ -0,0 +1,2171 @@
+
+First, we want to identify the logical names of your NIC interfaces. Connecting an [NVIDIA LinkX cable](https://www.nvidia.com/en-us/networking/interconnect/) in just one of the ports of the NIC will help you identify which port is which. Run the following command once the cable is in place:
+
+```bash
+ibdev2netdev
+```
+
+??? abstract "See an example output"
+
+    In the example below, only `mlx5_1` has a cable connected (`Up`), and its logical ethernet name is `eth1`:
+
+    ```bash
+    $ ibdev2netdev
+    mlx5_0 port 1 ==> eth0 (Down)
+    mlx5_1 port 1 ==> eth1 (Up)
+    ```
+
+??? failure "ibdev2netdev does not show the NIC"
+
+    If you have a cable connected but it does not show Up/Down in the output of `ibdev2netdev`, you can try to parse the output of `dmesg` instead. The example below shows that `0005:03:00.1` is plugged, and that it is associated with `eth1`:
+
+    ```sh
+    $ sudo dmesg | grep -w mlx5_core
+    ...
+    [   11.512808] mlx5_core 0005:03:00.0 eth0: Link down
+    [   11.640670] mlx5_core 0005:03:00.1 eth1: Link down
+    ...
+    [ 3712.267103] mlx5_core 0005:03:00.1: Port module event: module 1, Cable plugged
+    ```
+
+The next step is to set a static IP on the interface you'd like to use so you can refer to it in your Holoscan applications. First, check if you already have any addresses configured using the ethernet interface names identified above (in our case, `eth0` and `eth1`):
+
+```bash
+ip -f inet addr show eth0
+ip -f inet addr show eth1
+```
+
+If nothing appears, or you'd like to change the address, you can set an IP address through the Network Manager user interface, CLI (`nmcli`), or other IP configuration tools. In the example below, we configure the `eth0` interface with an address of `1.1.1.1/24`, and the `eth1` interface with an address of `2.2.2.2/24`.
+
+=== "One-time"
+
+    ```bash
+    sudo ip addr add 1.1.1.1/24 dev eth0
+    sudo ip addr add 2.2.2.2/24 dev eth1
+    ```
+
+=== "Persistent"
+
+    Set these variables to your desired values:
+
+    ```bash
+    if_name=eth0
+    if_static_ip=1.1.1.1/24
+    ```
+
+    === "NetworkManager"
+
+        Update the IP with `nmcli`:
+
+        ```bash
+        sudo nmcli connection modify $if_name ipv4.addresses $if_static_ip
+        sudo nmcli connection up $if_name
+        ```
+
+    === "systemd-networkd"
+
+        Create a network config file with the static IP:
+
+        ```bash
+        cat << EOF | sudo tee /etc/systemd/network/20-$if_name.network
+        [Match]
+        MACAddress=$(cat /sys/class/net/$if_name/address)
+
+        [Network]
+        Address=$if_static_ip
+        EOF
+        ```
+
+        Apply now:
+
+        ```bash
+        sudo systemctl restart systemd-networkd
+        ```
+
+!!! note
+
+    If you are connecting the NIC to another NIC with a [LinkX interconnect](https://www.nvidia.com/en-us/networking/interconnect/), do the same on the other system with an IP address on the same network segment.
```

Comment:
LinkX is a very specific term for something that's just "a cable". Do we want to leave a product-specific comment here when really any compatible cable/xcvr would work?

Reply:
agreed. I didn't know what term was best. Never heard of XCVR for example. I also heard of QSFP before then saw there was also OSFP. Should I use `SFP` ?

Reply:
xcvr is just shorthand for transceiver. I think saying "If you are connecting the NIC to another NIC directly with a cable instead of going through a switch, ..."